src/locales/de-CH/admin.ftl

```diff
@@ -0,0 +1,1426 @@
+### Localization for Admin

```

Comment:
since this is just copied from the english right now, it's better to exclude it and allow it to fall back to the english by default, so it doesn't get out of sync with the english translations.